src/components/Form/Form.tsx

```diff
@@ -452,9 +453,10 @@ const TextInputsGroup = ({
                               .join(" ")}
                             value={formState[inputId] as string}
                             onChange={(e, props) => {
-                              if (props?.value)
+                              if (props && "value" in props) {
                                 formState[inputId] = props.value.toString();
```

Comment:
Heads up, I'm not sure we actually want to `toString()` everything, but I left the existing behavior. Can talk about this outside of this PR.